Documentation/ContentElements/Media/Index.rst

```diff
@@ -1,44 +1,55 @@
-Embedding video and audio
-=========================
+================================================
+Embedding videos as content into a TYPO3 website
+================================================
 
 ..  image:: /Images/ManualScreenshots/ContentElements/TextAndMedia.png
     :alt: Content element Text and Media in the TYPO3 backend
     :class: img-thumbnail float-end ms-1
     :width: 250
 
-Like images, it's best practice to add new video and audio files using the
-:guilabel:`File > Filelist` module, which is covered in the :ref:`chapter about the file module <file-module>`. This method
-means files are stored centrally and any information or metadata you add to
-the file is used wherever that media file is used on the site.
+The content element "Text & Media" can be used in a similar fashion to the
+`Text & Images <https://docs.typo3.org/permalink/t3editors:images>`_ content
+element introduced in the previous chapter.
 
-When you use the :guilabel:`Select & upload files` button, the media file is
-attached to the page and gets uploaded to the :guilabel:`user_uploads` folder in the :guilabel:`Filelist`.
+It does however allow additional video and audio formats.
 
-.. tip::
-   By only using :guilabel:`Select & upload files`, your files pile up in the :guilabel:`user_uploads` folder pretty quickly.
-   Therefore you should rather upload and organize the files via the :guilabel:`Add Media` button or upload it with the :guilabel:`File > Filelist`
-   module directly, as described above.
+It can also be used to embed video players for YouTube and or Vimeo.
 
-.. _add-video-to-page:
+Which formats are allowed depends on the set up of your site. The allowed
+file extensions and pseudo formats (YouTube, Vimeo) are listed bellow the
+element selector.
+
+This content element can also be used to embed images and audio files.
+
+..  tip::
+    There is a large number of third party TYPO3 extensions to display a video
+    player or embed a video from an external source. If one of those is used
+    refer to the manual of the extension in use.
+
+..  _add-video-to-page:
 
 Add a video to a page
 =====================
 
-On the :guilabel:`Media` tab, click the :guilabel:`Add media file` button, then follow
-the same process as you would to :ref:`add-image-to-page`.
+If you want to embed a previously uploaded video or upload a new video use the
+button :guilabel:`Add media file`. If you want to embed a video from an external
+source like YouTube or Vimeo, use the button :guilabel:`Add media URL` instead.
 
-.. include:: /Images/AutomaticScreenshots/EditContent/EditContentMediaTab.rst.txt
+..  figure:: /Images/ManualScreenshots/ContentElements/Media.png
+    :alt: Tab media of the text and media content element in TYPO3
 
-Alternatively, you can click the :guilabel:`Add media by URL` button to paste
-a link to a video or audio file from the web.
+..  note::
+    For security reasons the maximal upload size is limited. Images can have
```

Comment:
"security"? I would say: To prevent exceeding your webspace limitations...

Reply:
Oki, so let us write "technical security" then?"

Reply:
Changed it to technical security